### ISSUE TITLE
Unsupervised Deformable Ultrasound Image Registration and Its Application for Vessel Segmentation. (arXiv:2306.13329v1 [eess.IV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#26577]
Unsupervised Deformable Ultrasound Image Registration and Its Application for Vessel Segmentation. (arXiv:2306.13329v1 [eess.IV])<br>
https://ift.tt/1SO7c6D<br>
<p>This paper presents a deep-learning model for deformable registration of
ultrasound images at online rates, which we call U-RAFT. As its name suggests,
U-RAFT is based on RAFT, a convolutional neural network for estimating optical
flow. U-RAFT, however, can be trained in an unsupervised manner and can
generate synthetic images for training vessel segmentation models. We propose
and compare the registration quality of different loss functions for training
U-RAFT. We also show how our approach, together with a robot performing
force-controlled scans, can be used to generate synthetic deformed images to
significantly expand the size of a femoral vessel segmentation training dataset
without the need for additional manual labeling. We validate our approach on
both a silicone human tissue phantom as well as on in-vivo porcine images. We
show that U-RAFT generates synthetic ultrasound images with 98% and 81%
structural similarity index measure (SSIM) to the real ultrasound images for
the phantom and porcine datasets, respectively. We also demonstrate that
synthetic deformed images from U-RAFT can be used as a data augmentation
technique for vessel segmentation models to improve intersection-over-union
(IoU) segmentation performance
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
